package/android/src/main/java/com/reactnativecommunity/slider/ReactSlider.java

```diff
@@ -66,6 +66,18 @@ public class ReactSlider extends AppCompatSeekBar {
 
   private List<String> mAccessibilityIncrements;
 
+  /** Real limit value based on min and max values. This comes from props */
+  private double mRealLowerLimit = Long.MIN_VALUE;
+
+  /** Limit based on progress from 0..100 */
+  private int mLowerLimit = 0;
```

Comment:
The reason is that the SeekBar's progress is an int. On `onProgressChanged` we want to set the progress to the limit value if it goes above or below the limits. I'm now thinking if we need `mRealLowerLimit` and `mRealUpperLimit` to be double since it will never get there. I initially set them as `double` to match `mMinValue` and `mMaxValue` which are double.